README.md

```diff
@@ -202,6 +204,25 @@ Result:
 }
 ```
 
+#### `addDeprecation(version, message)`
+
+Adds a deprecation on the request object allowing to notify when a controller or an action is deprecated for a specific version of Kuzzle
```

Comment:
A notice should be added explaining that the deprecation message shall only be visible in development environments:

```suggestion
Adds a deprecation on the request object allowing to notify when a controller or an action is deprecated for a specific version of Kuzzle

::: info
Deprecation messages in request responses will only be added during development stages, i.e. if the `NODE_ENV` environment variable is set to `development`.
:::
```